Minutes/2023/2023_08_07-minutes.md

```diff
@@ -0,0 +1,211 @@
+Meeting Minutes of 07 August 2023
+===
+
+:date: **Date:** 07 August 2023
+
+### :bust_in_silhouette: Participants
+
+- Ege Korkan
+- Cristiano Aguzzi
+- Fady Salama
+- Justus Fries
+- Idil Sezgin
+
+
+:writing_hand: **Scribe:** Ege Korkan
+
+### Tutorial Series
+
+Ege Korkan: we can talk about the tutorial series and then on the TPAC
+
+Ege Korkan: I have sent slides from the what is wot meetup, did you get them?
+
+Justus Fries: yes it worked
+
+Ege Korkan: Our original idea was to revamp the lecture series on the web of things.
+
+Ege Korkan: but that was very long, so we had to change the format and where we make it public.
+
+Ege Korkan: the storyline is based on the what you need to know in general about the web of things.
+
+Ege Korkan: We should also talk about semantic technologies which we were not doing at all.
+
+Ege Korkan: Preliminaries and then go into the web of things.
+
+Ege Korkan: We are putting some guidelines on things like the colors and like how to do this kind of animations.
+
+Ege Korkan: in essence we sketch in a very quick PowerPoint.
+
+Ege Korkan: then we found this and voice over generator. You can basically give the text and like choose even emotions and so on. realistic and audio output, but that you can easily work with the text and like generate the the video and adjust the timing accordingly.
+
+Ege Korkan: that's sort of workflow and we are recently starting on a like a full website.
+
+Ege Korkan: So that the the tutorial content can be read at the same time so that it's not just the the videos you have to watch like by stopping what you're doing  and so on and so forth.
```

Comment:
```suggestion
Ege Korkan: So that the tutorial content can be read and viewed at the same time. It's not just the the videos you have to watch and stopping the video to understand what you are doing.
```